src/applications/financial-status-report/sass/financial-status-report.scss

```diff
@@ -162,6 +162,11 @@
   margin: 20px 0;
 }
 
+// confirmation page
+.help-container {
```

Comment:
nit pick but could probably using a margin bottom class for this

Reply:
nope not in this case already uses margin this is just to tweak the confirmation page to match the other footers